plugins/start.py

```diff
@@ -5,9 +5,9 @@
 async def start(client, message):
     # return
     joinButton = InlineKeyboardMarkup([
-        [InlineKeyboardButton("Channel", url="https://t.me/aryan_bots")],
+        [InlineKeyboardButton("Movies", url="https://t.me/Mouryac_bot")],
         [InlineKeyboardButton(
-            "Report Bugs 😊", url="https://t.me/aryanvikash")]
+            "Report Bugs 😊", url="https://t.me/IAmMourya8")]
```

Comment:
If you're forking for making your own bot with the code, don't pull request to the original repo, commit to your own forked repo